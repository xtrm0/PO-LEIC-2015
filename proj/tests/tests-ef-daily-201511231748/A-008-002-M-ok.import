Objects in Space
Luke Skywalker/luke@jedi.com|Obi-Wan Kenobi/ob1@tatooine.geocities.com
SECTION||A Walk in the Desert
SECTION||Patience You Must Have
SECTION||Drama in the Clouds
