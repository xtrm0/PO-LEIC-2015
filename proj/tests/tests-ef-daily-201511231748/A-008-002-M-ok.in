5
Palpatine
pal@blackforce.com
4
0
