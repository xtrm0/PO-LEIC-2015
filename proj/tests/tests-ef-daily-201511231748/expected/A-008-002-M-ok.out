Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: Objects in Space
Autor: Luke Skywalker/luke@jedi.com
Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
Autor: Palpatine/pal@blackforce.com
Secções de topo: 3
Dimensão do documento (bytes): 77
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 